packages/web/docs/src/components/landing-page.tsx

```diff
@@ -154,7 +153,7 @@ function EnterpriseFocusedCards({ className }: { className?: string }) {
       className={cn('bg-beige-100 rounded-3xl px-4 pt-6 sm:py-24 md:px-6 md:py-[120px]', className)}
     >
       <Heading as="h3" size="md" className="text-balance sm:px-6 sm:text-center">
-        Enterprise-focused tooling at your disposal
+        Enterprise-focused tools tailored for you.
```

Comment:
Just FYI, one card has a full stop `.`, the next one exclamation mark `!`, and the text for Single Sign-On just ends without any interpunction.

Reply:
good point, I will add `?` to Single Sign-On to follow the rule :D

Reply:
exactly what I envisioned <3